test/10-file.t

```diff
@@ -196,6 +196,10 @@ is $batch->[3]{task},       'fail',       'right task';
 is $batch->[3]{state},      'finished',   'right state';
 is $batch->[3]{retries},    0,            'job has not been retried';
 ok !$batch->[4], 'no more results';
+# Minion::Command::minion::job calls list_jobs this way
+$batch = $minion->backend->list_jobs(0, 10, {queue => undef, state => undef,
```

Comment:
I'm not sure this is a valid invocation.  Can you provide an example of "minion::job" that breaks without the patch?  ie There must be a 'job' commandline that illustrates the breakage.